src/tools/setup-virtualenv.sh

```diff
@@ -22,12 +22,12 @@ virtualenv --python python2.7 $DIR
 . $DIR/bin/activate
 # older versions of pip will not install wrap_console scripts
 # when using wheel packages
-pip --log $DIR/log.txt install --upgrade 'pip >= 6.1'
+pip --disable-pip-version-check --log $DIR/log.txt install --upgrade 'pip >= 6.1'
```

Comment:
pip 1.5.4. it's shipped by ubuntu trusty. and see https://github.com/pypa/pip/commit/c3df28c7745af243f0c2e6f89af5bf2a93cbca60. the commit introducing this option was added after pip 6.0.